init_postgres.sql

```diff
@@ -0,0 +1,15 @@
+# docker run --rm --name postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 postgres -c wal_level=logical
+CREATE DATABASE gcom;
```

Comment:
I need to figure out how to init the Docker container with a database. I think we can mount the SQL actually.